scripts/parse_s19.pl

```diff
@@ -1,20 +1,10 @@
 #!/usr/bin/perl -w
-#
-# Copyright (C) 2018-2019 ETH Zurich and University of Bologna
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
+# Copyright 2023 ETH Zurich and University of Bologna.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 #
+# Yvan Tortorella <yvan.tortorella@unibo.it>
```

Comment:
are you sure it's only you? this is an old file..

Reply:
I fear this was a copy-paste error.